buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -441,43 +446,47 @@ def get_summary_table(identifier) {
         buildReleases.each { release ->
             // table cell
             def innerTable = "<table><tbody>"
+            def downstreamJobNames = buildFile.get_downstream_job_names(spec, release, identifier)
+            def downstreamBuilds = [:]
+            def pipelineName = ''
+            def build = null
```

Comment:
I don't think `build` needs to be initialized here. AFAICT it's set on L459 and only used on L461 and within that `if` block.

Reply:
Same comment for `pipelineName`

Reply:
removed